webapp/channels/src/components/sidebar_right/sidebar_right.tsx

```diff
@@ -228,11 +234,18 @@ export default class SidebarRight extends React.PureComponent<Props, State> {
         let content = null;
 
         if (postRightVisible) {
+            let fromSuppressed = false;
+            if (!this.lastOpenState && this.props.isOpen && this.lastSuppressedState) {
+                fromSuppressed = true;
+            }
             selectedChannelNeeded = true;
             content = (
                 <div className='post-right__container'>
                     <FileUploadOverlay overlayType='right'/>
-                    <RhsThread previousRhsState={previousRhsState}/>
+                    <RhsThread
+                        previousRhsState={previousRhsState}
+                        fromSuppressed={fromSuppressed}
```

Comment:
Nit: simplify and remove the unnecessary `let` above
```suggestion
                        fromSuppressed={!this.lastOpenState && this.props.isOpen && this.lastSuppressedState}
```